k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -390,6 +390,10 @@ spec:
                     description: GCEServiceAccount specifies the service account with
                       which the GCE VM runs
                     type: string
+                  gcpAliasRange:
+                    description: GCPAliasRange: Alias range from the pods CIDR that will be 
+                      assigned to the instances, defaults to /24
+                    type: string
```

Comment:
I'm not familiar with how strict the schema is and if you can convert types later, but I have question / idea.

Does using `integer` here simplifying things at all?  If you leave this as a `string`, validation will require that you strip the `/`, convert to `int`, then do a bounds check on the number of bits in the mask to enforce that it's within a reasonable range.  If you start with an `integer` field, you can skip the first two steps and jump right to using comparison operators, then simply `fmt.Sprintf("/%d", variableNameHere)` to get the string representation later.

Reply:
Building on this, we _might_ be able to use OpenAPI v3 schema validations and defaulting to do everything for us (the unquoted colon in the description looks a bit off too)  Something like:
```yaml
              gcpAliasRange:
                description: Alias range from the pods CIDR that will be
                  assigned to the instances, defaults to 24
                type: integer
                minimum: 24
                maximum: 28
                default: 24
```

Reply:
Doesn't seem these annotations actually work.   I've added validation to https://github.com/Invoca/kops/blob/c6fce4eafcbe40f416de95a183910fc8354c85f7/pkg/apis/kops/validation/gce.go